temporalio/bridge/src/lib.rs

```diff
@@ -0,0 +1,296 @@
+use pyo3::exceptions::{PyRuntimeError, PyValueError};
+use pyo3::prelude::*;
+use std::collections::HashMap;
+use std::time::Duration;
+use temporal_client::WorkflowService;
+use tonic;
+
+#[pymodule]
+fn temporal_sdk_bridge(_py: Python, m: &PyModule) -> PyResult<()> {
+    m.add_class::<ClientRef>()?;
+    m.add_function(wrap_pyfunction!(new_client, m)?)?;
+    Ok(())
+}
+
+type Client = temporal_client::RetryGateway<
+    temporal_client::ConfiguredClient<temporal_client::WorkflowServiceClientWithMetrics>,
+>;
+
```

Comment:
Probably makes sense to put everything in here in a client mod file, since this top-level will end up having actual bridge stuff

Reply:
Yeah, I figured when I get to worker stuff I will refactor to multiple files.